packages/blade/src/components/Typography/BaseText/BaseText.web.tsx

```diff
@@ -84,6 +87,7 @@ export const BaseText = ({
       textAlign={textAlign}
       numberOfLines={truncateAfterLines}
       wordBreak={wordBreak}
+      opacity={opacity}
```

Comment:
I don't think we should expose opacity. This will make base text colors unpredictable and will always rely on the background color. Lets speak with the design team once to confirm this

Reply:
As consulted with design, we have now increased the opacity to 64%.